src/index.ts

```diff
@@ -138,3 +139,54 @@ export function useLiveAnchorAccount<
     slotUpdated,
   };
 }
+
+function fetcher<I extends Idl, A extends keyof IdlAccounts<I>>(
+  program: Program<I>,
+) {
+  return function (method: A, ...params: [Address]): IdlAccounts<I>[A] {
```

Comment:
```suggestion
  return function (method: A, address: Address): IdlAccounts<I>[A] {
```

dont think we will be needing more params other than `address: Address`. Should we change it to just a 2-arg function instead?

Reply:
In that case yes, that makes sense